dev/Expander/Expander.xaml

```diff
@@ -157,6 +156,8 @@
                                 MinHeight="{TemplateBinding MinHeight}"
                                 MinWidth="{TemplateBinding MinWidth}"
                                 MaxWidth="{TemplateBinding MaxWidth}"
+                                HorizontalAlignment="Stretch"
```

Comment:
this needs to be template bound to the horizontal and vertical content alignment that you set above. Can't be hardcoded like this

Reply:
This control should be ignored for layout purposes and size to whatever the parent gives it or whatever the child content presenter needs. ContentAlignment is for the ContentPresenter ONLY.

Reply:
Also when/if the Expander supports Right/Left directions the MinWidth/MinHeight will have to be set by the visual state. It might be prudent to move this logic there now.

Reply:
Agreed.